File: medplum-java/medplum-java-model/src/main/java/com/fhir/CapabilityStatement_Implementation.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableCapabilityStatement_Implementation.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface CapabilityStatement_Implementation {
  Optional<List<Extension>> modifierExtension();

  Optional<String> id();

  Optional<List<Extension>> extension();

  Optional<String> description();

  Optional<Reference> custodian();

  Optional<url> url();

  static ImmutableCapabilityStatement_Implementation.Builder builder() {
    return ImmutableCapabilityStatement_Implementation.builder();
  }
}

File: medplum-typescript/src/model/fhir/CapabilityStatement_Implementation.ts
import { Extension } from "./Extension";
import { Reference } from "./Reference";
import { url } from "./url";

export interface CapabilityStatement_Implementation {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    description: string | null | undefined;
    url: url | null | undefined;
    custodian: Reference | null | undefined;
}
